### PR TITLE
Adding the regular functionality of Quit

### PR DESCRIPTION
**Pull request by mmatera**

### CODE REVIEW

File: mathics/builtin/assignment.py [hunk 1]
```diff
@@ -981,20 +981,29 @@ class Quit(Builtin):
...
+     = 3
+    >> Quit;
     >> a
      = a
 
     'Quit' even removes the definitions of protected and locked symbols:
     >> x = 5;
     >> Attributes[x] = {Locked, Protected};
-    >> Quit[]
+    >> Quit[];
```
Comment (sn6uv) [inline mathics/builtin/assignment.py]: Is there a reason for adding the extra semicolons here?

Comment (mmatera) [inline mathics/builtin/assignment.py]: No, of course... It is just that I use to end lines with semicolons when I do not look for output. I think that in this context would be better to do not use it, in order to see all the output.

**Comment by nabulator** [comment]
Python throws error when quits. 

<block>
Traceback (most recent call last):
  File "/usr/local/bin/mathics", line 9, in <module>
    load_entry_point('Mathics==0.8', 'console_scripts', 'mathics')()
  File "/usr/local/lib/python2.7/dist-packages/Mathics-0.8-py2.7.egg/mathics/main.py", line 306, in main
    shell.get_in_prompt(continued=total_input != ''))
  File "/usr/local/lib/python2.7/dist-packages/Mathics-0.8-py2.7.egg/mathics/main.py", line 95, in get_in_prompt
    next_line_number = self.get_last_line_number() + 1
  File "/usr/local/lib/python2.7/dist-packages/Mathics-0.8-py2.7.egg/mathics/main.py", line 91, in get_last_line_number
    line = self.definitions.get_definition('$Line').ownvalues[0].replace
IndexError: list index out of range
</block>

**Comment by mmatera** [comment]
Yes, but is the same error that you get if you call Quit[] in the main branch...

**Pull request merged**